simulator/src/utils.js

```diff
@@ -268,6 +270,24 @@ export function setupBitConvertor() {
             x = parseInt(inp, 2);
         setBaseValues(x);
     })
+    $("#bcdInput").on('keyup', function () {
+        var inp = $("#bcdInput").val();
+        var x = 0;
```

Comment:
Avoid single character variables

Reply:
Thanks for the suggestion sir,
Replaced variable name `x` with `num`.